### PR TITLE
fix: [CDS-71165]: Removed hardcoded automation-cdc org value from api/artifacts/azureartifacts/projects API query param

### PR DESCRIPTION
**Pull request by pankajsingh-harness**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by pankajsingh-harness** [comment]
Jest coverage is failing with:
```
Jest: "global" coverage threshold for statements (65%) not met: 24.54%
Jest: "global" coverage threshold for branches (50%) not met: 0%
Jest: "global" coverage threshold for lines (65%) not met: 24.54%
Jest: "global" coverage threshold for functions (50%) not met: 2.38%
```
Screenshot 2023-06-05 at 8 10 09 PM

**Review by satyam-harness — APPROVED** [review]

**Review by ishantmehtaharness — APPROVED** [review]
looks good
added a comment

File: src/modules/72-triggers/components/steps/ArtifactTriggerConfigPanel/ArtifactsSelection/ArtifactRepository/ArtifactLastSteps/AzureArtifacts/AzureArtifacts.tsx [hunk 1]
```diff
@@ -88,8 +88,7 @@ function FormComponent(
     lazy: true,
     queryParams: {
       ...commonParams,
-      connectorRef: connectorRefValue?.toString(),
-      org: 'automation-cdc'
```
[the remark this answers is not in the document]
Comment (pankajsingh-harness) [reply src/modules/72-triggers/components/steps/ArtifactTriggerConfigPanel/ArtifactsSelection/ArtifactRepository/ArtifactLastSteps/AzureArtifacts/AzureArtifacts.tsx:92]: BE has added `org` as required query param my mistake but not using this in code. So this has to be passed by UI to pass the Typescript check. cc: @satyam-harness

**Review by sonaligoyal-harness — APPROVED** [review]

**Comment by pankajsingh-harness** [comment]
Jest Coverage is failing with:
```
Jest: "global" coverage threshold for statements (65%) not met: 24.54%
Jest: "global" coverage threshold for branches (50%) not met: 0%
Jest: "global" coverage threshold for lines (65%) not met: 24.54%
Jest: "global" coverage threshold for functions (50%) not met: 2.38%
```
Screenshot 2023-06-06 at 10 33 17 AM

**Pull request merged**